corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -569,6 +569,11 @@ hqDefine("cloudcare/js/form_entry/fullform-ui", function () {
                     return options.data ? md.render(options.data) : null;
                 },
             },
+            help: {
+                update: function (options) {
+                    return options.data ? md.render(options.data) : null;
```

Comment:
Hey @knguyen142,
The changes you have made seem to be mostly safe but we are directly binding HTML in ko template for variable `help`. Can we sanitize `options.data` before assigning it to variable `help`.
```suggestion
                    return options.data ? md.render(DOMPurify.sanitize(options.data)) : null;
```


I am not sure if it will cause issues with markdown syntax that we support, you might need to test that.